packages/service-utils/src/core/authorize/types.ts

```diff
@@ -12,3 +12,13 @@ export type AuthorizationResult =
       errorMessage: string;
       errorCode: string;
     };
+
+export type RateLimitResult =
```

Comment:
This type is repeated below in rateLimit/types.ts
We don't need this one here right?